concepts.tex

```diff
@@ -527,14 +533,42 @@
 \ednote{Remove table [equalitycomparable] in [utility.arg.requirements]. Replace references to
 [equalitycomparable] with [concepts.lib.compare.equalitycomparable].}
 
+{\color{newclr}
+\indexlibrary{\idxcode{BikeshedEqualityComparable}}%
+\begin{itemdecl}
+template <class T, class U>
+concept bool BikeshedEqualityComparable() {
+  return requires(const T t, const U u) {
+    { t == u } -> Boolean;
+    { u == t } -> Boolean;
+    { t != u } -> Boolean;
+    { u != t } -> Boolean;
+  };
+}
+\end{itemdecl}
+
+\begin{itemdescr}
+Let \tcode{t} and \tcode{u} be objects of types \tcode{T} and \tcode{U}.
```

Comment:
Cut'n'pasted from the other EC concepts. I was considering changing them to "values" instead of "objects," but decided it can wait until I decide how to handle the "value type" concepts uniformly. As currently specified, `int(double, char)` satisfies `EqualityComparable`, and `EqualityComparable<int[42],int[]>()` is satisfied as well. The whole mess is detailed in #155.